app/src/test/java/nerd/tuxmobil/fahrplan/congress/schedule/LecturesByRoomTest.kt

```diff
@@ -0,0 +1,75 @@
+package nerd.tuxmobil.fahrplan.congress.schedule
+
+import com.google.common.truth.Truth.assertThat
+import nerd.tuxmobil.fahrplan.congress.models.Lecture
+import org.junit.Test
+
+class LecturesByRoomTest {
+
+    @Test
+    fun `no lectures`() {
+        val lecturesByRoom = LecturesByRoom(emptyList())
+
+        assertThat(lecturesByRoom).isEmpty()
+    }
+
+    @Test
+    fun `one lecture in room zero`() {
+        val roomZeroLecture = Lecture("").apply { roomIndex = 0 }
+        val lecturesByRoom = LecturesByRoom(listOf(roomZeroLecture))
+
+        assertThat(lecturesByRoom).hasSize(1)
+        assertThat(lecturesByRoom[0]).containsExactly(roomZeroLecture)
+    }
+
+    @Test
+    fun `one lecture in room index 1 and 3 - creates extract empty rooms for index 0 and 2`() {
```

Comment:
:abc: Something is mixed up here: `creates extract empty rooms for index 0 and 2`